uiowa/uiowa
# UI Press Book content type first draft

Resolves #4328 
Resolves #4329 

How to test:
1. blt ds  --site=uipress.uiowa.edu
2. Enable UI Press_Core module (drush @uipress.local en uipress_core)
3. drush -l uipress.uiowa.edu config:import --source ../config/sites/uipress.uiowa.edu --partial
5. drush @uipress.local cr
6. Create a book content type
7. Add some fields
8. Check sale logic by entering price and/or dates
9. If the sale price isn't added or if you're not within the sale date range, sale dates/price/code should not display. 
10. Test the opposite condition.
